app/controllers/api/v2/registration_command_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Api
+  module V2
+    class RegistrationCommandController < V2::BaseController
```

Comment:
**Just an idea**, I do not insist on this :)
I know `RegistrationCommand` is not a resource, but wouldn't it be better to treat it as one and thus name the controller in **plural** form `RegistrationCommandsController` ?
I'm just worried Rails could do unexpected things somewhere if we don't follow that pattern.
Let me know what you think :)